spec/cypress/integration/admin/InductionCoordinatorManagment.feature

```diff
@@ -0,0 +1,24 @@
+Feature: Admin user creating induction tutor
+  Admin user should be able to create an induction coordinator
+
+  Background:
+    Given I am logged in as an "admin"
+    And scenario "school_with_local_authority" has been run
+    And I am on "admin schools" page
+
+  Scenario: Create an induction tutor
+    When I click on "link" containing "Test school"
+    Then I should be on "admin school overview" page
+    And "page body" should contain "Test school"
+    And the page should be accessible
```

Comment:
Do we want some percy snapshots?

Reply:
When I put one in, the response was "Percy is not running", should I still put them in?

Reply:
Oh, when you were running it locally? Hm, I don't know what to expect locally, @callumacrae might know. 

This is branched off of something that has percy set up, right?

Reply:
I will put one in and see what happens...